pyrobots/src/App.js

```diff
@@ -43,12 +44,12 @@ function App() {
         {/* A <Routes> looks through its children <Route>s and
             renders the first one that matches the current URL. */}
         <Routes>
-          <Route path="/" element={<Home/>}/>
-          <Route path="/create_user" element={<Create_user/>}/>
-          <Route path="/login" element={<Login/>}/>
-          <Route path="/list_matches" element={<List_matches/>}/>
-          <Route path="/create_match" element={<Create_match/>}/>
-          <Route path="/create_robot" element={<Create_robot/>}/>
+          <Route path="/" element={<Home />} />
+          <Route path="/create_user" element={<RequireToken><Create_user /></RequireToken>} />
```

Comment:
I think that for create_user page token is not required.

Reply:
> `SyntaxError: Cannot use import statement outside a module`

I tried to change the syntax of the axios module import but the error kept appearing. The way I did it is the same as in the axios documentation.